changelog.dd

```diff
@@ -28,12 +28,12 @@ $(BUGSTITLE Library Changes,
         exposed))
     $(LI $(RELATIVE_LINK2 iota-length-size_t, `std.range.iota's `.length`
         property is fixed to `size_t` instead of the type being iterated))
-    $(LI $(XREF uni, isNumber) and $(XREF uni, isPunctuation) now use a separate,
+    $(LI $(REF isNumber, std,uni) and $(REF isPunctuation, std,uni) now use a separate,
```

Comment:
AFAIK it's also on @CyberShadow's list to compile the Phobos changelog with DAutoTest (though if i am not mistaken he wanted to wait for #4228).
Hopefully we can merge #4228 soon and editing the changelog will become easier.
Ideally there should be no need to preview.